apps/backend/src/modules/meetings/meeting.service.ts

```diff
@@ -0,0 +1,107 @@
+import { HTTPCode } from "~/libs/modules/http/http.js";
```

Comment:
Change the file name to `meetings.service.ts` to have the same naming approach everywhere